spec/features/contextual_navigation_spec.rb

```diff
@@ -325,7 +325,7 @@ def given_there_is_a_non_step_by_step_parent_page
   end
 
   def not_step_by_step_content
-    not_step_by_step_content = random_item("placeholder")
+    not_step_by_step_content = random_item("generic")
```

Comment:
Not a blocker, but I feel like we might end up having a similar problem with "generic" at some point. The only [example in publishing-api](https://github.com/alphagov/publishing-api/blob/main/content_schemas/examples/generic/frontend/services_and_information.json#L10-L12) lists "whitehall-frontend" as the rendering app, which is no longer true of anything.

Reply:
Good point 👍 I've switched all the references to `guide` now